### ISSUE TITLE
how to restore my trained snapshopts to resume my training?

### ISSUE THREAD

**Issue by galaxy-fangfang** [eldar/pose-tensorflow#57]
the iterations is 1800000, which i cant complete it in one time, i want to know how to restore my trained parameters?
i see your code:
sess.run(tf.global_variables_initializer())
    sess.run(tf.local_variables_initializer())

    # Restore variables from disk.
    restorer.restore(sess, cfg.init_weights)

but when i change the cfg.init.weights to snapshots-610000, the loss is as high as the beginning of my first training.

**Comment by chaiban** [comment]
Hi @galaxy-fangfang , did you figure it out ?. can training be resumed ? 
Help me with the code if possible.
Thank you.
